mvlearn/embed/kcca.py

```diff
@@ -49,10 +50,14 @@ class KCCA(BaseEmbed):
           Regularization parameter
     decomp : string, default = 'full'
              Decomposition type
```

Comment:
add something saying at least saying something like "Full or Incomplete Cholesky Decomposition: ICD is used to reduce computation times"